server/conf/i18n/messages

```diff
@@ -152,23 +152,33 @@ button.select=Select
 button.startApp=Start an application
 # Button to view the applications belonging to an applicant.
 button.viewApplications=View applications
+# Button to view and add clients
+button.viewAndAddClients=View and add clients
 
 # Part of the confirmation banner which appears after a new client has been added to a community-based organization.
 content.clientCreated=Your new client {0} has been added to your client list. You can now begin submitting applications for this client.
+# The tooltip of the email input field
+content.emailTooltip=Add an email address for your client to receive status updates about their application automatically. Without an email, you or your community-based organization will be responsible for communicating updates to your client
 # For each client on the client list, shows the number of applications that have been submitted for them.
 content.numberOfAppSubmitted={0} applications submitted
+# Shows the one application that have been submitted for them.
+content.oneAppSubmitted=1 application submitted
 # Some explanation text on the confirmation modal when deleting a client.
 content.warning=There are no applications submitted for this client, so no applications will be lost.
+# Shows there is no email address connected to the applicant.
```

Comment:
suggestion (blocking):  Since this is also used in the TI organization members table, we might say "Shows there is no email address connected to the account."

Reply:
changed `applicant` to `account`.